testing-the-tester.js

```diff
@@ -0,0 +1,96 @@
+
+var grid = []
+
+function restart () {
+// It should restart the game so it can be played again.
+// It is assumed that the turns of the player will be automatically changed after an allowed move.
+// The application will console log all the passed or failed test.
+  if (isGameOver === true) {
+    grid = []
+  }
+}
+
+function isGameOver(){
+//It should return a true or false if the game is over.
+  var status = whoWon()
+  if (status === 0) {
+    return false
+  }
+  else if ((status === 1) || (status === 2)) {
+    return true
+  }
+  else if (status === 3) {
+    return true
+  }
+}
+
+function whoWon(){
+// It should return 0 if the game is not yet finished. Else it should return
+// either 1 or 2 depending on which player one. It should return 3 if the game is a draw.
+  if (grid.length === 9) {
+    return 3
+  }
+  //Splitting grid into each player array
+  var player1 = []
+  for (var i = 0; i < grid.length; i+2){
```

Comment:
Interesting logic to add the player into the even number array, but i think it's not necessary. You can just iterating through the single grid array.